### PR TITLE
Add Redis Housekeeping

### LINKED ISSUE #79: Add Redis Housekeeping

### PR DESCRIPTION
**Pull request by exp625**
Fixes #79

### CODE REVIEW

**Comment by exp625** [comment]
This was easier than I thought

File: pkg/cache/redis.go [hunk 1]
```diff
@@ -20,21 +20,21 @@ var (
 
 const (
 	defaultKeyPrefix = "cache"
-	defaultMaxAge    = 30
+	defaultTTL       = 60 * 60 * 24 // Cache entries should update every day
```
Comment (DrJoLe) [inline pkg/cache/redis.go:23]: Please use `go fmt`

File: api/store/redis.go [hunk 2]
```diff
@@ -21,7 +21,8 @@ import (
 
 const (
 	cookieKeyLength            = 32
-	cookieMaxAge               = 7 * 24 * 60 * 60
+	cookieMaxAge			   = 60 * 60 * 24 * 7 // Cookie can last for 7 days
```
Comment (DrJoLe) [inline api/store/redis.go:24]: Please use `go fmt`

**Review by DrJoLe — CHANGES_REQUESTED** [review]
See comments

File: api/store/redis.go (hunk 2 above)
Comment (DrJoLe) [inline api/store/redis.go:24]: Please use `go fmt`

File: pkg/cache/redis.go (hunk 1 above)
Comment (DrJoLe) [inline pkg/cache/redis.go:23]: Please use `go fmt`

**Review by DrJoLe — CHANGES_REQUESTED** [review]

**Review by DrJoLe — APPROVED** [review]
Thanks

**Pull request merged**